crates/router/src/routes/admin.rs

```diff
@@ -206,10 +206,7 @@ pub async fn delete_merchant_account(
 /// Merchant Connector - Create
```

Comment:
Nit: We can remove the doc comments here, since they're anyway being read from the `openapi` crate for the purpose of the OpenAPI documentation. Can be taken up in a separate PR.

Reply:
Created an issue for this, will take this up in next PR - https://github.com/juspay/hyperswitch-cloud/issues/6606